Adyen/Service/PosPaymentLocalApi.cs

```diff
@@ -86,6 +86,20 @@ public SaleToPOIResponse TerminalApiLocal(SaleToPOIMessage saleToPoiRequest, Enc
         {
             return TerminalApiLocal(saleToPoiRequest: saleToPoiRequest, encryptionCredentialDetails: encryptionCredentialDetails);
         }
+        
+        /// <summary>
+        /// Used to decrypt the notification received
+        /// </summary>
+        /// <param name="notification"></param>
+        /// <param name="encryptionCredentialDetails"></param>
+        /// <returns></returns>
+        public string DecryptNotification(string notification, EncryptionCredentialDetails encryptionCredentialDetails)
+        {
+            var saleToPoiMessageSecured = _saleToPoiMessageSecuredSerializer.Deserialize(notification);
+            var decryptNotification = _messageSecuredEncryptor.Decrypt(saleToPoiMessageSecured, encryptionCredentialDetails);
+            this.Client.LogLine("Notofication: \n" + decryptNotification);
```

Comment:
```suggestion
```

Just talked with @AlexandrosMor and we think it's better to remove this log entry. It doesn't look like a good idea to log a decrypted message.